Index: index.html
===================================================================
diff --git a/index.html b/index.html
deleted file mode 100644
--- a/index.html	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,38 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Multiplication Table</title>
-    <link rel="stylesheet" type="text/css" href="stylesheet.css">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-</head>
-<body>
-
-<section class="container">
-    <h1>Multiplication Table Project &#129451;</h1>
-    <h3>Create your own multiplication table that is rendered with an HTML table!</h3>
-
-    <div class="generate_box">
-        <div class="inputs">
-            <label for="rows">Rows:
-                <input id="rows" type="number" value="12" placeholder="How many rows?"/>
-            </label>
-
-            <label for="cols">Columns:
-                <input id="cols" type="number" value="12" placeholder="How many columns?"/>
-            </label>
-
-            <label for="highlight">Highlights:
-                <input id="highlight" type="number"  value="" placeholder="What's highlighted?"/>
-            </label>
-        </div>
-
-        <button id="generate" onclick="creatwithhtmltable(getinputvalueasnumber('rows'), getinputvalueasnumber('cols'), getinputvalueasnumber('highlight'))">Generate Table!</button>
-    </div>
-
-    <div id="output"></div>
-</section>
-
-</body>
-<script src="javascript.js"></script>
-</html>
\ No newline at end of file
Index: package.json
===================================================================
diff --git a/package.json b/package.json
deleted file mode 100644
--- a/package.json	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,10 +0,0 @@
-{
-  "name": "project_2",
-  "version": "1.0.0",
-  "description": "",
-  "main": "javascript.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "private": true
-}
Index: javascript.js
===================================================================
diff --git a/javascript.js b/javascript.js
deleted file mode 100644
--- a/javascript.js	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,100 +0,0 @@
-/* function getrowsdesired() {
-    console.log(document.getElementById('rows').value);
-    return Number(document.getElementById('rows').value);
-}
-
-function getcolumnsdesired() {
-    console.log(document.getElementById('cols').value);
-    return Number(document.getElementById('cols').value);
-}
-
-function gethihglightvalue() {
-    let highlightvalue = document.getElementById('highlight').value; */
-
-/* This method replaces 3 methods above */
-function getinputvalueasnumber (inputID) {
-    if (document.getElementById(inputID)) {
-    if (document.getElementById(inputID).value) {
-        return Number(document.getElementById(inputID).value)
-    }
-} else {
-    console.log('Sorry, couldn\'t find this ID:', inputID)
-    }
-}
-
-function creatwithhtmltable(rows, cols, highlight) {
-    let string
-    if (rows && cols) {
-        string = "<table>\n"
-
-        for (let row =1; row <= rows; row++) {
-            if (highlight && highlight === row) {
-                string += "<tr class='highlight'>\n"
-            } else {
-                string += "<tr>\n"
-            }
-
-            for (let col =1; col <= cols; col++) {
-
-                if (highlight && highlight === col) {
-                    string += "<td class='highlight'>\n"
-                } else {
-                    string += "<td>"
-                }
-
-                string += row * col
-                string += "</td>\n"
-            }
-            string += "</tr>\n"
-        }
-        string += "</table>\n"
-    } else {
-        string = 'Provide some inputs'
-    }
-    document.getElementById("output").innerHTML = string;
-}
-
-/* Loop Examples
-console.log('here\'s a loop for showing 1...10:\n')
-for (let i = 0; i < 10; i++) {
-    console.log(`${ i }\n`)
-}
-
-console.log('here\'s a d0-while for showing 1...10:\n')
-let i = 0
-do {
-    console.log(i + `\n`)
-    i++
-} while (i<10)
-
-console.log('here\'s a while for showing 1...10:\n')
-let wil = 0
-while (wil < 10) {
-    console.log(`${ wil }\n`)
-    wil++
-}
-
-console.log('here\'s a foreach loop iterating over an array showing 1...10\n')
-let nums =[...Array(10).keys()]
-nums.forEach(num => {
-    console.log(`${ num }\n`)
-})
-
-console.log('Here\'s a multiplication table method rendered in the console with a for loop:\n')
-console.log('This demos variables, concatenation, for loops, console.logging and methods, along with method paragraph')
-let timestable = (to) => {
-    console.log('Here\'s a times table for to ' + to + '\n')
-
-    if (Number.isInteger(to)) {
-        for (let i = 1; i <= to; i++) {
-            let row = ""
-            for (let j = 1; j <= to; j++) {
-                row += `${i * j}\t`
-            }
-            console.log(row)
-        }
-    } else {
-        console.log(`sorry, ${to} isn't a number!`)
-    }
-}
-timestable(12)*/
\ No newline at end of file
Index: forms_review.html
===================================================================
diff --git a/forms_review.html b/forms_review.html
deleted file mode 100644
--- a/forms_review.html	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,29 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Forms Review</title>
-
-    <style type="text/css">
-        table{
-            td{
-
-            }
-        }
-    </style>
-</head>
-<body>
-
-<label for="age">
-        <p>What's your age?</p>
-        <p>We need this to calculate your date of birth!</p>
-</label>
-<input type="number" id="age">
-
- <label for="name">
-     <p>What's your name?</p>
- </label>
-<input type="text" id="name">
-
-</body>
-</html>
\ No newline at end of file
Index: stylesheet.css
===================================================================
diff --git a/stylesheet.css b/stylesheet.css
deleted file mode 100644
--- a/stylesheet.css	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,63 +0,0 @@
-section.container {
-    width:  70vw;
-    height: 100vh;
-    margin: 0 auto;
-
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    gap: 10px;
-    font-family: Avenir, Helvetica, Arial, sans-serif;
-
-    .generate_box {
-        display: flex;
-        gap: 10px;
-
-        .inputs {
-            display: flex;
-            flex-direction: column;
-            gap: 5px;
-
-            label {
-                display: flex;
-                gap: 10px;
-                justify-content: space-between;
-            }
-        }
-        button {
-            background-color: deepskyblue;
-        }
-    }
-
-    #output {
-        overflow: scroll;
-        max-width: 90vw;
-        max-height: 60vh;
-
-        table, th, td {
-            border: 2px solid black;
-            border-collapse: collapse;
-            padding: 4px;
-        }
-
-        table {
-            border: 4px solid black;
-
-            tr:first-of-type, tr:last-of-type, td:first-of-type, td:last-of-type {
-                background-color: cornflowerblue;
-            }
-
-            .highlight {
-                background-color: goldenrod;
-            }
-
-            td {
-                padding: 2px;
-                text-align: center;
-            }
-        }
-        tr:hover {
-            background-color: lightcoral;
-        }
-    }
-}
\ No newline at end of file
Index: table_review.html
===================================================================
diff --git a/table_review.html b/table_review.html
deleted file mode 100644
--- a/table_review.html	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
+++ /dev/null	(revision 794a020b18cac4a51acdc73ba88fa048f1a7a00f)
@@ -1,43 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Tables Review</title>
-
-    <style type="text/css">
-        table{
-            border: 1px solid black;
-
-            th, td{
-              border: 1px solid #ddd;
-            }
-        }
-    </style>
-</head>
-<body>
-
-<table>
-    <tr>
-        <th>Name</th>
-        <th>Age</th>
-        <th>City</th>
-        <th>State</th>
-    </tr>
-
-    <tr>
-        <td>Wil</td>
-        <td>20</td>
-        <td>Chicago</td>
-        <td>IL</td>
-    </tr>
-
-    <tr>
-        <td>Alex</td>
-        <td>20</td>
-        <td>Minneapolis</td>
-        <td>MN</td>
-    </tr>
-</table>
-
-</body>
-</html>
\ No newline at end of file
